### DISCUSSION TITLE
Replace axios with unfetch?

### DISCUSSION THREAD

**Discussion by tbgse** [react-static/react-static#1487] [General]
We're trying to keep our bundle size as small as possible, but since the library is relying on axios in various places, that makes it impossible for us to get rid of it in the vendor bundle. Have you considered to move this to [unfetch](https://github.com/developit/unfetch) and make the bundle even smaller? Without having too much insight into the library itself, I wonder if this is something that could be done without too much effort?

**Comment by SleeplessByte** [comment]
Just checked:

`axios` _can_ be replaced. We would need [isomorphic-unfetch](https://github.com/developit/unfetch/tree/master/packages/isomorphic-unfetch) instead, so it also works during SSR.
